cypress/e2e/specialFeatures/atiAnalytics/assertions/index.js

```diff
@@ -5,6 +5,9 @@ import {
   interceptATIAnalyticsBeacons,
 } from '../helpers';
 
+const usesReverbViewabilityModel = ({ useReverb, appEnv }) =>
```

Comment:
As mentioned (above? below?), let's not add `appEnv` as a param here - let's just pass in `useReverb`, and check whether `getAppEnv() !== 'live'`?